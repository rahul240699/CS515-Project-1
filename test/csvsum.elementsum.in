python3 prog/csvsum.py salary_data.csv YearsExperience Salary -s