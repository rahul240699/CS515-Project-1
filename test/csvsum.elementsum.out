0      39344.1
1      46206.3
2      37732.5
3      43527.0
4      39893.2
5      56644.9
6      60153.0
7      54448.2
8      64448.2
9      57192.7
10     63221.9
11     55798.0
12     56961.0
13     57085.1
14     61115.5
15     67942.9
16     66034.1
17     83093.3
18     81368.9
19     93946.0
20     91744.8
21     98280.1
22    101309.9
23    113820.2
24    109439.7
25    105591.0
26    116978.5
27    112644.6
28    122401.3
29    121882.5
dtype: float64